integration_test/third_party_apps_data/applications/postgresql/metadata.yaml

```diff
@@ -31,31 +31,145 @@ minimum_supported_agent_version:
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.blocks_read
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      database: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.commits
     value_type: INT64
-    kind: CUMULATIVE
+  - kind: GAUGE
     monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.connection.max
+    value_type: INT64
+  - kind: GAUGE
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.database.count
+    value_type: INT64
+  - kind: GAUGE
     labels:
       database: .*
-  - type: workload.googleapis.com/postgresql.db_size
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.db_size
     value_type: INT64
-    kind: GAUGE
+  - kind: CUMULATIVE
+    labels:
+      database: .*
+      table: .*
+      index: .*
     monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.index.scans
+    value_type: INT64
+  - kind: GAUGE
     labels:
       database: .*
-  - type: workload.googleapis.com/postgresql.rollbacks
+      table: .*
+      index: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.index.size
     value_type: INT64
-    kind: CUMULATIVE
+  - kind: CUMULATIVE
+    labels:
+      database: .*
+      table: .*
+      operation: ins|upd|del|hot_upd
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.operations
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      replication_client: .*
     monitored_resource: gce_instance
+    optional: true
+    type: workload.googleapis.com/postgresql.replication.data_delay
+    value_type: INT64
+  - kind: CUMULATIVE
     labels:
       database: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.rollbacks
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      database: .*
+      state: dead|live
+      table: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.rows
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      database: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.table.count
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      database: .*
+      table: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.table.size
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      database: .*
+      table: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/postgresql.table.vacuum.count
+    value_type: INT64
+  - kind: GAUGE
+    monitored_resource: gce_instance
+    optional: true
+    type: workload.googleapis.com/postgresql.wal.age
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      operation: flush|replay|write
+      replication_client: .*
```

Comment:
The scope [doc](https://docs.google.com/document/d/1M9r8CqZl2W9fYz8brzlWntEAqJUC1K43LHGvGkbpiPs/edit) does not include a `replication_client` label.

Is this an intentional change to the scope doc? If so, can we send a suggested edit to the scope doc and have the approval double check it? If not, can we fix the translation so it matches the scope doc?

Reply:
This was intentional as to properly identify the metric. See https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-REPLICATION-VIEW for more information on the table.

Will send a suggestion